radix-engine/src/engine/kernel.rs

```diff
@@ -717,6 +720,7 @@ where
                         }
                     }
                 };
+                drop(current_frame);
```

Comment:
I think better to use rust scoping to implement this drop. e.g.:
```
let node_pointer = {
    let current_frame = Self::current_frame(&self.call_frames);
    if current_frame.owned_heap_nodes.contains_key(&node_id) {
        RENodePointer::Heap {
            frame_id: current_frame.depth,
            root: node_id.clone(),
            id: None,
         }
    } else if let Some(pointer) = current_frame.node_refs.get(&node_id) {
          pointer.clone()
     } else {
         match node_id {
             // Let these be globally accessible for now
             // TODO: Remove when references cleaned up
             RENodeId::ResourceManager(..) | RENodeId::System => {
                 RENodePointer::Store(*node_id)
             }
             _ => {
                 return Err(RuntimeError::KernelError(
                     KernelError::InvokeMethodInvalidReceiver(*node_id),
                 ))
             }
          }
      };
}
```

Reply:
Good idea. Will apply